src/controllers/busControllers.js

```diff
@@ -0,0 +1,72 @@
+import models from '../database/models';
+
+const createBus = async (req, res) => {
+  try {
+    const bus = await models.Bus.create(req.body);
+    return res.status(201).json({ status: 201, message: 'Bus created successfully', bus });
+  } catch (error) {
+    return res.status(500).json({ error: error.message });
+  }
+};
+
+const getAllBuses = async (req, res) => {
+  try {
+    const buses = await models.Bus.findAll();
+    if (buses.length < 1) { return res.status(404).json({ status: 404, message: 'There are no available buses' }); }
+    return res.status(200).json({ buses });
+  } catch (error) {
+    return res.status(500).send(error.message);
+  }
+};
+
+const getBusById = async (req, res) => {
+  try {
+    const { busId } = req.params;
+    const bus = await models.Bus.findOne({
+      where: { id: busId },
+    });
+    if (bus) {
+      return res.status(200).json({ bus });
+    }
+    return res.status(404).json({ status: 404, message: 'Bus with the specified ID does not exist' });
+  } catch (error) {
+    return res.status(500).send(error.message);
+  }
+};
+
+const updateBus = async (req, res) => {
+  try {
+    const { busId } = req.params;
+    const [updated] = await models.Bus.update(req.body, {
+      where: { id: busId },
+    });
+    if (updated) {
+      const updatedBus = await models.Bus.findOne({ where: { id: busId } });
+      return res.status(200).json({ status: 200, message: 'Bus was updated successfully.', bus: updatedBus });
+    }
+    return res.status(404).json({ status: 404, message: `Cannot update Bus with id ${busId}. Bus was not found!` });
```

Comment:
It would be enough to just say: Bus with ID (id) was not found

Reply:
The issue is now fixed